### PR TITLE
moderation and logging

### PR DESCRIPTION
**Pull request by TheMoksej**
I hope I didn't mess up anything whilst doing this pr

### CODE REVIEW

**Comment by TheMoksej** [comment]
mk looks like I didn't

**Comment by TheMoksej** [comment]
Anyways I've added kick and ban commands that work properly, you can change the format though

**Comment by Cairo2k18** [comment]
oh ty i'll update stuff later on just keep this as a draft ig

File: cogs/moderation.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                for num, res in enumerate(success_list, start=0):
+                    muted += f"`[{num+1}]` {res}\n"
+                for num, res in enumerate(failed_list, start=0):
+                    notmuted += f"`[{num+1}]` {res}\n"
+                await ctx.send(muted + notmuted)
+            if not success_list and failed_list:  
+                notmuted += f"**I failed to ban all the members:**\n"
+                for num, res in enumerate(failed_list, start=0):
+                    notmuted += f"`[{num+1}]` {res}\n"
+                await ctx.send(notmuted)
```
Comment (DuckMasterAl) [inline cogs/moderation.py:144]: Yes, notmuted and muted. Very nice.

Comment (TheMoksej) [reply cogs/moderation.py:144]: I'd change the variables but stupid github doesn't let me do that

**Comment by TheMoksej** [comment]
I will be closing this since I don't want there to be 3 unnecessary branches, will introduce this in the rewrite.

**Pull request closed without merging**